### ISSUE TITLE
css selector

### ISSUE THREAD

**Issue by li7721994** [Imangazaliev/DiDOM#55]
There is a problem.
Please look at the code as follow
```PHP
$html = <<<HTML
<select name="sort-field" class="sort_field">
    <option selected="selected" value="/en/tsuk/category/brands-4210405/adidas/N-7z2Z21o4Zdgl?Nrpp=20&amp;siteId=%2F12556&amp;sort_field=Relevance">Best Match</option>
    <option value="/en/tsuk/category/brands-4210405/adidas/N-7z2Z21o4Zdgl?Nrpp=20&amp;Ns=product.freshnessRank%7C0&amp;siteId=%2F12556&amp;sort_field=Newness">Newest</option>
</select>
HTML;

$Document = new DiDom\Document;
$Document->loadHtml($html);

$Element = $Document->first('select[name=sort-field]  option[selected=selected]'); // find nothing 
$Element = $Document->first('select[name=sort-field]')->first('option[selected=selected]'); // it's ok

```

**Comment by Imangazaliev** [comment]
Thank you for your feedback. I found a bug in the parser (rather in the CSS to XPath compiler) and now think how to solve it.

**Comment by Imangazaliev** [comment]
Fixed in [1.8.6](https://github.com/Imangazaliev/DiDOM/releases/tag/1.8.6)
